src/__tests__/loadLinks.test..ts

```diff
@@ -0,0 +1,8 @@
+import {  loadLinks } from '../index';
+
+describe('Tests for LoadLinks Function', () => {
+  test('result from "loadLinks" is not null', async () => {
+    const links = await loadLinks({});
```

Comment:
since this function makes an external API call, testing it in this way will mean that this test is testing both the function and the API. If the API call fails or the server goes down for some reason, will `loadLinks` return `null` and cause the test to fail even if nothing about the function has changed?

I think it might be better to set up a mock for AsyncAirtable so that it returns predefined responses that would allow us to test how the function handles both success and failure responses without actually making any API calls